cvat-core/src/api-implementation.ts

```diff
@@ -189,6 +190,38 @@ export default function implementAPI(cvat: CVATCore): CVATCore {
         return users;
     });
 
+    implementationMixin(cvat.apiTokens.get, async (filter) => {
+        checkFilter(filter, {
+            id: isInteger,
+            page: isInteger,
+            page_size: isInteger,
+            filter: isString,
+            sort: isString,
+            search: isString,
+            name: isString,
+            owner: isInteger,
+            read_only: isBoolean,
+            created_date: isString,
+            updated_date: isString,
+            expiry_date: isString,
+            last_used_date: isString,
+        });
+
+        const result = await serverProxy.apiTokens.get(filter);
+        const tokens = result.map((tokenData) => new ApiToken(tokenData));
+        Object.assign(tokens, { count: result.count });
+        return tokens as PaginatedResource<ApiToken>;
+    });
+
+    implementationMixin(cvat.apiTokens.update, async (id, tokenData) => {
+        const result = await serverProxy.apiTokens.update(id, tokenData);
+        return new ApiToken(result);
+    });
+
+    implementationMixin(cvat.apiTokens.revoke, async (id) => {
+        await serverProxy.apiTokens.revoke(id);
+    });
```

Comment:
I believe `update` and `revoke` should be implemented as methods of ApiToken class. 
I see you already call `serverProxy::update` from `save` method.